blockchain.go

```diff
@@ -207,6 +210,7 @@ var defaultConfig = func() config {
 		Logger:                       zerolog.Nop(),
 		TransactionValidationEnabled: true,
 		ChainID:                      flowgo.Emulator,
+		CoverageReportingEnabled:     true,
```

Comment:
Should this be enabled by default?

Reply:
Not really, it's up for discussion. On another topic, have you had a look on this https://github.com/onflow/flow-emulator/pull/332#issuecomment-1471840314? I am struggling to find the cause of this, so we can contain the changes in this repository, and discard the `flow-go` companion PR.

Reply:
Updated default value to `false`